### ISSUE TITLE
Select node values precisely

### ISSUE THREAD

**Issue by Justin-Johnson** [Justin-Johnson/ReJJ#7]
UI to allow typing of node value numbers.

Using keyboard won't work as it requires the JSFX to take full control of the keyboard which I don't want.

**Comment by m13253** [comment]
What about a floating screen keyboard?
(Similar to MeldaProduction's VST design.)
<img width="514" alt="image" src="https://user-images.githubusercontent.com/1808930/191424771-805590ce-99f7-4028-a127-16259c33780f.png">

**Comment by Justin-Johnson** [comment]
Yes, it's occurred to me - think I might have mentioned it on the Reaper JSFX forum.  It's a good idea.
